adapters/smaato/smaato.go

```diff
@@ -187,20 +188,29 @@ func (adapter *adapter) makeIndividualRequests(request *openrtb2.BidRequest) ([]
 }
 
 func splitImpressionsByMediaType(imp *openrtb2.Imp) ([]openrtb2.Imp, error) {
-	if imp.Banner == nil && imp.Video == nil {
-		return nil, &errortypes.BadInput{Message: "Invalid MediaType. Smaato only supports Banner and Video."}
+	if imp.Banner == nil && imp.Video == nil && imp.Native == nil {
+		return nil, &errortypes.BadInput{Message: "Invalid MediaType. Smaato only supports Banner, Video and Native."}
 	}
 
-	imps := make([]openrtb2.Imp, 0, 2)
+	imps := make([]openrtb2.Imp, 0, 3)
```

Comment:
I am pondering if this should be `make([]openrtb2.Imp, 0 ,3*len(imp))`, as the maximum number of imps you could see would be the orginal number of imps times 3, which would be the case if each imp was multiformat with all three allowed formats. Or more likely just 2, as I don't expect you would see a multiformat imp with all three of banner, video, and native very often, if at all.